Form/Type/AdminType.php

```diff
@@ -46,7 +46,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 $options['delete_options']['type_options']['translation_domain'] = $admin->getTranslationDomain();
             }
 
-            $builder->add('_delete', $options['delete_options']['type'], $options['delete_options']['type_options']);
+            $builder->add(
+                '_delete',
+                $options['delete_options']['type'],
+                array_merge(
+                    $options['delete_options']['type_options'],
+                    array('attr' => array(
+                        'class' => 'sonata-admin-type-delete-checkbox', ),
```

Comment:
Hi,
My guess its done by `trailing_comma_in_multiline_array` rule.
Looking at the code there are 2 multi line arrays, both starting on line 54, one ending on 55, other on 56. Therefor the fixer will add ',' at the end of each element. Guess this is what it makes look odd. 
You could, for example, update the current;

```php
array('attr' => array(
    'class' => 'sonata-admin-type-delete-checkbox', ),
)
```
to:

```php
array(
    'attr' => array( 'class' => 'sonata-admin-type-delete-checkbox' )
)
```

but you can define the style yourselves of course :)
As I see it, its not a bug in the fixer, it just does what you asked it to do.